scripts/puppetserver.sh

```diff
@@ -12,6 +12,9 @@ SERVICE_STOP_RETRIES=60
 # ExecStartPre
 /usr/bin/install --directory --owner=puppet --group=puppet --mode=775 /var/run/puppetlabs/puppetserver
 
+# set ownership on var directory
+chown puppet /var/lib/puppet
```

Comment:
Can you update this to be:

```
chown -R puppet:puppet /var/lib/puppet/
```

We want it to be owned user and group `puppet` recursively.